components/brave_rewards/core/mojom/rewards_database.mojom

```diff
@@ -71,3 +71,17 @@ struct DBCommandResponse {
   DBCommandResult? result;
   Status status;
 };
+
+interface RewardsDatabase {
+  RunDBTransaction(DBTransaction transaction) => (DBCommandResponse response);
+};
+
+struct HashPrefixData {
+  string prefixes;
+  uint32 prefix_size;
+};
+
+interface HashPrefixStore {
```

Comment:
Do we use actually use this interface for IPC? If not why just not use sequenceBound?

Reply:
I considered passing this interface to the Rewards engine, but decided to leave that for a future improvement. In the meantime, it is used to run the interface on a different thread (via `RemoteWorker`). However, if you aren't comfortable with that approach, I can use `SequenceBound` instead.

Reply:
Ok, it's up to you.
From my perspective it adds extra complexity, but could be useful in some scenarios.

Reply:
I decided to go with `SequenceBound` - thanks for the suggestion.